app/services/apiQueues.js

```diff
@@ -12,12 +12,30 @@ function queues(queueFactory) {
 }
 
 function exec(message) {
-  request(message.options, function(error, response, body){
-    if (error){
-      console.log(error);
-      message._reject(error);
+  const {
+    options: {
+      method,
+      url
+    }
+  } = message;
+
+  fetch(url, message.options).then(function(response) {
```

Comment:
Importing a big dataset (a session with 42000 files total raw size 648 MB) gives me this error:
![screen shot 2016-11-21 at 11 34 31 am](https://cloud.githubusercontent.com/assets/1427389/20497621/b439a8a2-afde-11e6-82ae-44abb9e0eaca.png)

Not sure it is related to performance, but maybe some size limits have been hit.